utils/gpii-net.tf

```diff
@@ -0,0 +1,717 @@
+### A ###
```

Comment:
Should not this be `aws/utils`, since it is dealing with AWS resources?

Reply:
Right, I made this utility before the new repository structure